src/object/class_object.c

```diff
@@ -2183,6 +2183,28 @@ classobj_has_class_unique_constraint (SM_CLASS_CONSTRAINT * constraints)
   return false;
 }
 
+/*
+ * classobj_has_class_rk_constraint ()
+ *   return: true if a replication key constraint is contained in the constraint list,
+ *           otherwise false.
+ *   constraints(in): constraint list
+ */
+bool
+classobj_has_class_rk_constraint (SM_CLASS_CONSTRAINT * constraints)
```

Comment:
추후 관련 코드를 키워드로 찾기 쉽도록, 전체적인 구현 용어를 통일하면 좋을 것 같습니다. rk_, replication_key_, repl_key_, rkey_ 등이 후보군이 될 수 있을 것 같습니다.
```suggestion
classobj_has_class_repl_key_constraint (SM_CLASS_CONSTRAINT * constraints)
```

Reply:
@hornetmj  일단 classobj_has_class_repl_key_constraint() 로 반영하였습니다. 키워드에대해서는 대략적으로 기능과 함수들의 구현 반영하도록하겠습니다